.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,96 @@
+name: New Issue
+description: Create a new issue
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
+         server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
+      3. Please take a moment to check that your issue doesn't already exist.
+      4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
+      5. Please give all relevant information below for bug reports, because
+         incomplete details will be handled as an invalid report.
+- type: input
+  id: gitea-ver
+  attributes:
+    label: Gitea Version
+    description: Gitea version (or commit reference) your instance is running
+    placeholder: v1.14.3
+  validations:
+    required: true
+- type: input
+  id: git-ver
+  attributes:
+    label: Git Version
+    description: The version of git running on the server's systemm
+  validations:
+    required: true
```

Comment:
I don't think the git version should be required.
Until now, most of the issues I opened could be managed without knowing the git version. Additionally, doing it that way disables sending bugs present on try.gitea.io as I have no idea what git version is running there.

Reply:
I see your point there, in my next commit this will be patched.